src/mgr/BaseMgrModule.cc

```diff
@@ -139,6 +139,9 @@ ceph_send_command(BaseMgrModule *self, PyObject *args)
 
   MonCommandCompletion *command_c = new MonCommandCompletion(self->py_modules,
       completion, tag, PyThreadState_Get());
+
+  PyThreadState *tstate = PyEval_SaveThread();
```

Comment:
after nautilus, we could have a `WithoutGIL`  scoped lock (or mutex) where we don't need to hold GIL. just like how we use `Gil` where GIL is required.

Reply:
yes! i was thinking the same thing